### DISCUSSION TITLE
[Request] BunkerWeb WAF

### DISCUSSION THREAD

**Discussion by LeoSpyke** [tteck/Proxmox#3034] [Ideas]
Hi everybody, the BunkerWeb project would be a nice to have in a catalogue like this.

For those who don't know it yet, it is basically an NGINX reverse proxy with tons of useful built-in security features like WAF, bots and bad IPs blocking capability, HTTPS certificates handling via Let's Encrypt, DNSBL and many more. A great alternative to grant a secure as possible access to our self-hosted services from outside IMHO.

It could be added into the Server-Networking category near to Caddy and Nginx Proxy Manager.

**Comment by tteck** [comment]
https://tteck.github.io/Proxmox/#bunkerweb-lxc

**Comment by mgrimace** [comment]
Out of curiosity @LeoSpyke how does this 'protect' other LXCs? Does it have to replace my existing reverse proxy (e.g., NPM)?

**Comment by Matze1116** [comment]
I get an error while installing:
![Screenshot 2024-07-30 074019](https://github.com/user-attachments/assets/462dde11-dc11-4942-b0b8-bc0889ae1365)

**Reply by tteck** [reply]
Fixed https://github.com/tteck/Proxmox/commit/d07353534663f87186eceaf8c4a76ce9e886e0df

**Comment by Matze1116** [comment]
Thx for the fast reply:Another error in Speedtest ->  - Installing MySpeed   [ERROR] in line 23: exit code 0: while executing command "$@" > /dev/null 2>&1The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.THXAm 30.07.2024 um 13:51 schrieb tteckster ***@***.***>:﻿
Fixed d073535
